src/commands/gitleaks.yaml

```diff
@@ -1,113 +1,28 @@
 parameters:
-  branch:
-    description: "The branch to audit."
-    type: "string"
-  commit:
-    description: "Scan all files changed by this commit (rev. SHA)."
-    type: "string"
-  commit-from:
-    description: "Commit to start audit from (rev. SHA). This is the most recent commit to audit. Default is the latest."
-    type: "string"
   commit-to:
     description: "Commit to stop audit (rev. SHA). This is the oldest commit to audit. Default is the root commit. \
     In CircleCI, you can use << pipeline.git.base_revision >> to scan only since the parent commit."
     type: "string"
   config:
     description: "Path to a config file to use for gitleaks configuration."
     type: "string"
```

Comment:
Why should we change the config between builds?

Reply:
For example, different rules for master vs branch builds

Reply:
Per branch changes we should be able to cover from inside the orb IMO.

Reply:
I'm still convinced we should expose some way to tune this, but I guess the alternative is assuming every repository will use a config file with the same path. This path must exist, so the orb could detect if it exists and change the command based on whether the file is there or not. Would that be better?

Reply:
We talked about this. The `config` param stays. We want to at least be able to whitelist files.